src/compiler/core.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path="performance.ts" />
 
 namespace ts {
+    // WARNING: The script `configureNightly.ts` uses a regexp to parse out these values.
+    // If changing the text in this section, be sure to test `configureNightly` too.

```

Comment:
That's a great idea! I'll try to work on it soon and we can compare likes for likes.